oauth2client/contrib/django_util/__init__.py

```diff
@@ -236,16 +313,47 @@ def __init__(self, settings_instance):
             _get_oauth2_client_id_and_secret(settings_instance)
 
         if ('django.contrib.sessions.middleware.SessionMiddleware'
-                not in settings_instance.MIDDLEWARE_CLASSES):
-            raise exceptions.ImproperlyConfigured(
-                "The Google OAuth2 Helper requires session middleware to "
-                "be installed. Edit your MIDDLEWARE_CLASSES setting"
-                " to include 'django.contrib.sessions.middleware."
-                "SessionMiddleware'.")
+           not in settings_instance.MIDDLEWARE_CLASSES):
+                raise exceptions.ImproperlyConfigured(
```

Comment:
Oh, I get it! It's five spaces here because the `not` on the previous line is only three spaces in from the `if` on the line above it.

I'm fine to live with whatever `flake8`, `pyformat`, and `yapf` produce.